yarn-project/Earthfile

```diff
@@ -238,10 +244,12 @@ scripts-prod:
     SAVE ARTIFACT /usr/src /usr/src
 
 all:
+    ARG RAYON_NUM_THREADS
     BUILD +aztec
     BUILD +end-to-end
 
 export-aztec:
+    ARG RAYON_NUM_THREADS
```

Comment:
Pretty sure this isn't necessary, args are basically env vars passed into the entire build that are opted info in each command that needs them